testautoRIFT.py

```diff
@@ -858,6 +858,79 @@ def generateAutoriftProduct(indir_m, indir_s, grid_location, init_offset, search
                     master_split = str.split(master_filename,'_')
                     slave_split = str.split(slave_filename,'_')
 
+#                    master_MTL_path = master_path[:-6]+'MTL.txt'
+#                    slave_MTL_path = slave_path[:-6]+'MTL.txt'
+#
+#                    master_time = str.split(str.split(runCmd('fgrep "SCENE_CENTER_TIME" '+master_MTL_path))[2][1:-2],':')
+#                    slave_time = str.split(str.split(runCmd('fgrep "SCENE_CENTER_TIME" '+slave_MTL_path))[2][1:-2],':')
+
+                    import netcdf_output as no
+                    pair_type = 'optical'
+                    detection_method = 'feature'
+                    coordinates = 'map'
+                    if np.sum(SEARCHLIMITX!=0)!=0:
+                        roi_valid_percentage = int(round(np.sum(CHIPSIZEX!=0)/np.sum(SEARCHLIMITX!=0)*1000.0))/1000
+                    else:
+                        raise Exception('Input search range is all zero everywhere, thus no search conducted')
+    #                out_nc_filename = 'Jakobshavn_opt.nc'
+                    PPP = roi_valid_percentage * 100
+                    if ncname is None:
+                        out_nc_filename = f"./{master_filename[0:-8]}_X_{slave_filename[0:-8]}" \
```

Comment:
@maricarag please change -8 to -7. This has been corrected in the [PR](https://github.com/nasa-jpl/autoRIFT/pull/44/files) I pushed yesterday. You can take a look. My PR will fix the L8 naming problem (if you remember the netcdf file name was missing a T number: "_T1_X_" is shown as "_T_X_").